src/raft/storage.rs

```diff
@@ -66,6 +90,7 @@ impl Default for MemStorageCore {
 }
 
 impl MemStorageCore {
+    // set_hardstate saves the current HardState.
```

Comment:
Use `///` for docs comment. See also https://doc.rust-lang.org/stable/book/documentation.html